modules/distribution/pom.xml

```diff
@@ -616,6 +616,12 @@
                                     <version>${siddhi.io.ibmmq.version}</version>
                                     <type>jar</type>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.wso2.extension.siddhi.io.cdc</groupId>
+                                    <artifactId>siddhi-io-cdc</artifactId>
+                                    <version>${siddhi.io.cdc.version}</version>
```

Comment:
Parent pom with version is not committed.

Reply:
Fixed with commit: 29bc2bde951f8c2cb19f17f599f918be88b48df5